pkg/server/catalog/catalog.go

```diff
@@ -155,8 +165,37 @@ type Repository struct {
 	catalog.Closer
 }
 
+// parsePortedUpstreamCA update configuration, parsing ported UpstreamCA plugins into UpstreamAuthority
+func parsePortedUpstreamCA(pluginConfig catalog.HCLPluginConfigMap, log logrus.FieldLogger) error {
```

Comment:
What do you think if we rename this function to "portUpstreamCA"? I don't see that the "parse" prefix fits well in this case.